modules/manage/pages/kubernetes/k-scale-redpanda.adoc

```diff
@@ -282,3 +293,9 @@ kubectl get persistentvolumeclaim --namespace <namespace>
 kubectl get persistentvolume --namespace <namespace>
 ----
 
+After the Nodewatcher controller has finished, xref:manage:kubernetes/k-decommission-brokers.adoc[decommission the broker] that was removed from the node. This is necessary to prevent quorum loss and ensure cluster stability.
```

Comment:
I was reviewing my knowledge of the ghost broker situation and there's a missing link in my knowledge about exactly how quorum loss happens. Would it be worth nuancing this with something like "potential loss of quorum" or "eventual loss of quorum"?